VertexCompositeProducer/src/LamC3PFitter.cc

```diff
@@ -61,6 +62,25 @@ float cand2Mass_sigma[2] = {protonMassLamC3P_sigma, piMassLamC3P_sigma};
 
 const double c_cm_ns = 2.99792458e1; //[cm/ns] // mtd
 
+inline float LamC3PFitter::invBetaPion(const float& p){
+    return std::sqrt(1 + std::pow(piMassLamC3P/p,2));
+}
+
+inline float LamC3PFitter::invBetaKaon(const float& p){
+    return std::sqrt(1 + std::pow(kaonMassLamC3P/p,2));
+}
+
+inline float LamC3PFitter::invBetaProton(const float& p){
+    return std::sqrt(1 + std::pow(protonMassLamC3P/p,2));
+}
+
+TF1* fExpPionBTL = new TF1("fExpPionBTL_dInvBetaRMS","0.005 + 0.016*exp(-x/4.4)", 0.7, 1000);
```

Comment:
Try to avoid using pointers and creating new objects globally that are not destroyed later.
Instead, simply create the object:
TF1 fExpPionBTL("fExpPionBTL_dInvBetaRMS","0.005 + 0.016*exp(-x/4.4)", 0.7, 1000);